src/subscription/data/service.js

```diff
@@ -0,0 +1,50 @@
+import { ensureConfig, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+import handleRequestError from '../../payment/data/handleRequestError';
+import { transformResults } from '../../payment/data/utils';
+
+ensureConfig([
+  'ECOMMERCE_BASE_URL',
+  'LMS_BASE_URL',
+], 'payment API service');
```

Comment:
Can you add a TODO to modify with correct API later?

Reply:
done.